src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="input input--large" data-qa="big">
+      <input
+        class="input__field input__field--large"
+        type="text"
+        name="big-search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </div>
```

Comment:
It's better to use semantic tags where possible. Instead of `div`, you can use `section` or `article` here. It will improve your page SEO and help screen readers. `div` does not have any meaning.